import-scripts/update-az-mskimpact.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+export AZ_DATA_HOME=$PORTAL_DATA_HOME/az-msk-impact-2022
+export AZ_MSK_IMPACT_DATA_HOME=$AZ_DATA_HOME/mskimpact
+export AZ_TMPDIR=$AZ_DATA_HOME/tmp
+
+source $PORTAL_HOME/scripts/dmp-import-vars-functions.sh
+
+# 1. Pull latest from AstraZeneca repo (mskcc/az-msk-impact-2022)
+printTimeStampedDataProcessingStepMessage "pull of AstraZeneca data updates to git repository"
+
+cd $AZ_DATA_HOME ; $GIT_BINARY pull origin
```

Comment:
the importer --fetch-data command results in
git pull origin <branchname>
git-lfs pull origin <branchname>

(at least for git repositories)